idaes/apps/grid_integration/docs/index.rst

```diff
@@ -13,29 +13,59 @@ Welcome to grid_nmpc's documentation!
-uncertainties, e.g., energy locational marginal prices (LMPs), weather, fuel
-prices, renewable generation. Any accurate forecasting method is compatible with
-the framework. These forecasts then facilitate the energy systems to compute optimal
-market bids under uncertainty. Energy bids are usually marginal production costs
-of energy systems, and they are submitted to the market operator and enter the objective
-functions of market-clearing problems. The day-ahead market (figure right) is cleared by
-solving unit commitment problems. Similarly, in the real-time market cycle
-(figure left), economic dispatch problems are solved every
-hour to clear the market and set generation targets for each resource. Each
-energy system then solves optimal control problems to track these dispatch signals from
-real-time markets constrained by (non)linear process models. Finally, the market
-calculates the settlements and pays the energy systems.
+wholesale electricity markets. This multiscale framework aims to:
+(1) elucidate complex relationships between resource dynamics and market dispatch
+(with uncertainty, beyond price-taker assumption);
+(2) predict the economic opportunities and market impacts of emerging
+technologies (e.g., H2 production, integrated energy systems);
+(3) guide conceptual design & retrofit to meet current and future power grid
+needs.
+
+The framework integrates optimal operations, and control of an energy system with
+market clearing via a high-fidelity Production Cost Model (PCM) Prescient which
+optimizes resource dispatch decisions across an entire transmission network.
+Specifically, in the day-ahead market loop, the following steps are modeled:
+(a) Forecast: energy systems perform market uncertainty forecasts, e.g., locational
+marginal prices (LMP).
+(b) Bid: energy systems utilize the uncertainty forecasts and solve optimization
+problems to derive optimal bidding strategies into the markets.
+(c) Clearing: after collecting day-ahead bids from market participants, the market
+operator (modeled by Prescient) solves unit commitment problems to clear the
+day-ahead market.
+Similarly, in the real-time market loop, the following steps are modeled:
+(i) Dispatch: after collecting hourly real-time bids from market participants,
+the market operator (modeled by Prescient) dispatches power generation schedules
+to energy systems by solving real-time economic dispatch problems.
+(ii) Tracking: energy systems solve optimal nonlinear model predictive control
+(NMPC) problems to track the dispatch signals.
+(iii) Settle: based on the actual energy production schedules, the market operator
+calculates and pays the energy and ancillary service settlements to the participants.
+
+.. |doubleloop_implementation| image:: images/implementation.png
+  :width: 400
+  :alt: Alternative text
+  :align: middle
+
+|doubleloop_implementation|
+
+The design of framework's implementation is shown in the figure above. Specifically,
```

Comment:
Replace the first two sentences with: "The above figure shows the implementation. The objects ``Forecaster``, ``Forecaster``, ``Bidder``, ``Tracker``, and ``Coordinator`` help manage information exchange within the multiscale integrated simulation framework.